src/main/scala/viper/gobra/ast/frontend/PrettyPrinter.scala

```diff
@@ -437,6 +437,16 @@ class DefaultPrettyPrinter extends PrettyPrinter with kiama.output.PrettyPrinter
     case PNamedOperand(id) => showId(id)
     case PBoolType() => "bool"
     case PIntType() => "int"
+    case PInt8Type() => "int8"
+    case PInt16Type() => "int16"
+    case PInt32Type() => "int32"
```

Comment:
For example here, the int32 could have been a rune, as well.

Reply:
Fixed in cd4d443